src/components/frontend-language-toggle/README.md

```diff
@@ -58,28 +58,28 @@ The package is owned by the DI Frontend Capability team, part of the development
 
 1. Install NPM package
    ```sh
-   npm install one-login-language-toggle
+   npm install language-toggle
```

Comment:
i think we need to keep 

when i run 
`npm install language-toggle` it says not found so i think we need to keep the org name